src/prelaunch_data_generator/el_cl_genesis/el_cl_genesis_generator.star

```diff
@@ -56,16 +56,22 @@ def generate_el_cl_genesis_data(
     )
 
     genesis = plan.run_sh(
-        run="cp /opt/values.env /config/values.env && ./entrypoint.sh all",
+        run="cp /opt/values.env /config/values.env && ./entrypoint.sh all && mkdir /network-configs && mv /data/custom_config_data/* /network-configs/",
```

Comment:
can we make changes to the genesis generator so that the generation is in line with what we expect from dencun?

(i.e not have the mv etc here); not now for later

Reply:
We do the mv here too lol: https://github.com/ethpandaops/ansible-collection-general/blob/master/roles/ethereum_genesis/tasks/generate_genesis.yaml#L75

Reply:
Changing the format upstream is a hassle, as lots of projects might potentially be using this. Id hold off with this unless its really needed.